.pre-commit-config.yaml

```diff
@@ -1,4 +1,12 @@
 repos:
+-   repo: https://github.com/PyCQA/autoflake
+    rev: v1.7.6
+    hooks:
+    -   id: autoflake
+        name: autoflake
+        args: ["--in-place", "--remove-unused-variables", "--remove-all-unused-imports"]
+        language: python
+        files: \.py$
```

Comment:
shouldn't need name or language or files here.  args might not be needed either, unclear on phone

Reply:
Thanks!

`args` however is needed:

> By default, autoflake only removes unused imports for modules that are part of the standard library.

Also without `in-place` it just prints the changes, instead of applying them.

https://pypi.org/project/autoflake/